_events/2023/2023-06-education-training-talk.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Machine Learning for your Research
+subtitle: US-RSE Education & Training Seminar Series
+expires: 2023-06-22
+event_date: "June 22, 2023"
+layout: event
+duration: 120
+repeated: false
+category: education-training
+time:
+    - - start: 2023-06-22T18:00:00Z
+        end: 2023-06-22T20:00:00Z
+---
+
+## US-RSE Education & Training Seminar Series
+
+US-RSE periodically presents technical talks and tutorials related to Education & Training for RSEs.
+
+## Event
+
+The next technical talk of the US-RSE Education & Training Seminar Series will feature Savannah Thais who will speak about "Machine Learning for your Research."
+This event will take place **Thursday, June 22nd at 2 PM ET, 1 PM CT, 12 PM MT, 11 AM PT**
+
+*Abstract*: This technical talk will give an overview of modern supervised and unsupervised machine learning (ML) methods. We will discuss the advantages and limitations of each and explore what types of problems each is best suited to address. We will also discuss best practices for using ML methods in scientific and social impact research including appropriate model evaluation/testing and reproducibility and generalizability considerations.
+
+*Learning Objectives*: Attendees will leave with an understanding of common ML algorithms, the types of data they require, and what types of problems they are best suited for. Attendees will also be presented with a framework for evaluating ML models within the context of their work. If time allows, we will spend time discussing and brainstorming specific project ideas from participants’ individual research.
+
+*Intended Audience*: This workshop will be most useful for people whose research has (or could have) at least some quantitative elements and who are interested in incorporating ML into their work. It might also be interesting for people not currently involved in such research but curious about how ML can be used in research more generally. Attendees should have a "big picture" concept of what ML entails, namely selecting an algorithm with a mathematically defined learning goal and then using data examples to adjust that algorithm's parameters in order to move towards this goal is very useful but not explicitly required as we will cover these topics at the beginning of the class. Participants should also have an understanding of what sorts of data exist in their field or project and what kinds of questions they might want to answer with ML
```

Comment:
Missing a period at the end of the paragraph

Reply:
I will add the period.